### ISSUE TITLE
Indicator Menu: Sound (feature request)

### ISSUE THREAD

**Issue by JassMan23** [ubports/ubuntu-touch#939]
### Description of the feature

One thing that seems to be missing from all smart phones is the ability to go silent only for a limited time. Would it be possible to add to the Sound dropdown on the Notification Menu, the ability to set a timer? ie below the Silent Mode switch, a timer selection (maybe hours & minutes spinners - which I actually hate but occupy the smallest space) so that silent mode returns to normal after a preset time. It could be either silent until a given time hh:mm  or for a selected interval. 

I am sure I am not the only one who forgets to turn ringing back on after watching a film or a play. 

All the series 40,60 and 70 Nokia chocolate bars had  "timed profiles" which even allowed you to change the wall paper, menu colours, etc., while in timed mode. In fact because it was all done with profiles, you could have a permanent vibrate only mode, and change to a timed ringer profile which returned to vibrate only, when the time was up. Because it was profile driven, you could have Loud, Quiet, Vibrate only, and totally silent choices and switch between them permanently or for a selected interval.
Smart phones aren't always an improvement., but we can always try.
 
### Illustrations

None

**Comment by Fuseteam** [comment]
I wonder if we go further and add profile sheduling
Hmm

**Comment by JassMan23** [comment]
Wow! that really would make it really useful. Say you were going to the theatre and were expecting a call. You could warn the caller that after the start will be too late, then schedule phone to go silent at the start. At the interval you could then check to see if the expected call had come in or not.
I have yet to be the one person who ignored the requests to switch off your phone then have it go off in the middle of a play but as I age an memory gets weaker, this would be a helpful backstop.

**Comment by Mraanderson** [comment]
Lineage OS has timed silent or alarms only. Scheduling for weekdays and weekends too. This is a very desirable feature.

**Comment by clendee** [comment]
I agree, this would be really a useful feature, definately something Ive been missing from the experience.
